elements/dhcp-on/install.d/95-dhcp-on

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Default to eno1
+NETIF=${DIB_DHCP_ON_DEV:-eno1}
```

Comment:
Do we still end up with an ifcfg for eth0 that is baked into the cloud image? Would it make sense to remove that if $DIB_DHCP_ON_DEV != eth0?